book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+

```

Comment:
- [ ] Kindly create a setter method for the `:rentals` variable. Ideally, a Book instance's rental array should be accessible since it shares a many-to-many relationship with the `Person` class instance(s). Having a setter method is ideal for this situation. Kindly implement this change. 👍 

#### Hint: Take a look at the way you correctly set it in the `Person` class